src/enigma2/data/RecordingEntry.h

```diff
@@ -106,6 +109,7 @@ namespace enigma2
       mutable bool m_radio = false;
       mutable bool m_haveChannelType = false;
       mutable bool m_anyChannelTimerSource = false;
+      mutable bool m_deleted = false;
```

Comment:
As far as I can see there is no need to flag `m_deleted` mutable as you do not modify it in any `const` member function.